tests/test_util.py

```
@@ -658,5 +658,645 @@ def test_consistency_across_languages(self):
+
+    @patch('phoonnx.util.is_numeric')
+    def test_normalize_number_word_complex_separator_handling(self, mock_is_numeric):
+        """Test complex separator handling in _normalize_number_word."""
+        # Test when both thousands and decimal separators are present
+        mock_is_numeric.side_effect = lambda x: x == "123456.78"
+        
+        with patch('phoonnx.util.pronounce_number') as mock_pronounce:
+            mock_pronounce.return_value = "one hundred twenty three thousand"
+            
+            # European format: 123.456,78
+            result = _normalize_number_word("123.456,78", "pt", None)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)